File: replace_ui.py
import os
import re

def process_dir(directory):
    for root, dirs, files in os.walk(directory):
        for file in files:
            if file.endswith(('.tsx', '.ts', '.jsx', '.js')):
                full_path = os.path.join(root, file)
                with open(full_path, 'r', encoding='utf-8') as f:
                    content = f.read()
                
                before = content
                
                # Remove all shadow-[...]
                content = re.sub(r'shadow-\[[^\]]+\]', 'shadow-sm', content)
                content = re.sub(r'hover:shadow-\[[^\]]+\]', 'hover:shadow-md', content)
                content = re.sub(r'active:shadow-\[[^\]]+\]', 'active:shadow-inner', content)
                
                # Replace neumorphic backgrounds
                content = re.sub(r'bg-\[#f0f0f3\]', 'bg-white', content)
                content = re.sub(r'bg-\[#cbced1\]', 'bg-gray-200', content)
                
                if content != before:
                    with open(full_path, 'w', encoding='utf-8') as f:
                        f.write(content)

process_dir('frontend/src')
print('Done replacement')

File: replace_ui.js
const fs = require('fs');
const path = require('path');

function processDir(dir) {
  const files = fs.readdirSync(dir);
  for (const file of files) {
    const fullPath = path.join(dir, file);
    if (fs.statSync(fullPath).isDirectory()) {
      processDir(fullPath);
    } else if (fullPath.match(/\.(tsx|ts|jsx|js)$/)) {
      let content = fs.readFileSync(fullPath, 'utf8');
      const before = content;
      
      // Remove all shadow-[...]
      content = content.replace(/shadow-\[[^\]]+\]/g, 'shadow-sm');
      content = content.replace(/hover:shadow-\[[^\]]+\]/g, 'hover:shadow-md');
      content = content.replace(/active:shadow-\[[^\]]+\]/g, 'active:shadow-inner');
      
      // Replace neumorphic backgrounds
      content = content.replace(/bg-\[#f0f0f3\]/g, 'bg-white');
      content = content.replace(/bg-\[#cbced1\]/g, 'bg-gray-200');
      
      if (content !== before) {
        fs.writeFileSync(fullPath, content);
      }
    }
  }
}

processDir('frontend/src');
console.log('Done replacement');
